### PR TITLE
Create Exceptionhandler & ExceptionHandlerTest classes

### PR DESCRIPTION
**Pull request by Krushit-Babariya**
### Exception Handling

**1. Exception Handling (ExceptionHandler class):**
The ExceptionHandler class is annotated with @ControllerAdvice, which allows it to handle exceptions globally for the entire application. It has handlers for the following exceptions:
_NoSuchElementException_: Returns a 404 Not Found response when an element, such as an employee, is not found.
_IllegalArgumentException_: Returns a 400 Bad Request when an invalid UUID or other bad input is encountered.
_MethodArgumentNotValidException_: Handles validation errors, capturing invalid input fields and their associated messages, returning a 400 Bad Request.
_Generic Exception_: Catches all other unhandled exceptions, returning a 500 Internal Server Error.

**2. Unit Testing (ExceptionHandlerTest class):**
The ExceptionHandlerTest class contains unit tests to ensure that each exception handler in the ExceptionHandler class behaves as expected. It tests:
Correct handling of NoSuchElementException, ensuring it returns a 404 status.
Handling of IllegalArgumentException, verifying a 400 status.
Proper validation error handling (MethodArgumentNotValidException) by mocking field validation and checking that it returns the appropriate errors.
A generic Exception handler that ensures unexpected errors result in a 500 Internal Server Error.

### CODE REVIEW

File: spring-boot-app-example/src/main/java/io/github/mfvanek/pg/cluster/exceptions/ExceptionHandler,java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package io.github.mfvanek.pg.cluster.exceptions;
```
Comment (mfvanek) [inline spring-boot-app-example/src/main/java/io/github/mfvanek/pg/cluster/exceptions/ExceptionHandler,java:1]: Misspelling in filename `ExceptionHandler,java`. Replace comma with period.

File: spring-boot-app-example/src/test/java/io/github/mfvanek/pg/cluster/exceptions/ExceptionHandlerTest.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+import org.springframework.web.bind.MethodArgumentNotValidException;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.NoSuchElementException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+public class ExceptionHandlerTest {
```
Comment (mfvanek) [inline spring-boot-app-example/src/test/java/io/github/mfvanek/pg/cluster/exceptions/ExceptionHandlerTest.java:17]: We need a kind of integration test here.
For example, make a request via webTestClient and got `handleInvalidUUIDFormat` error.

**Review by mfvanek — CHANGES_REQUESTED** [review]

File: spring-boot-app-example/src/test/java/io/github/mfvanek/pg/cluster/exceptions/ExceptionHandlerTest.java (hunk 2 above)
Comment (Krushit-Babariya) [reply spring-boot-app-example/src/test/java/io/github/mfvanek/pg/cluster/exceptions/ExceptionHandlerTest.java:17]: Done it

**Review by Krushit-Babariya — COMMENTED** [review]
Review Changes..!

**Pull request closed without merging**